docs/portal/developer-portal/advanced/Enabling_K3s.md

```diff
@@ -30,8 +30,51 @@ The following steps should be completed prior to configuring the UAN with K3s.
    **Note**: In a future release, additional UANs will be able to join as extra manager or worker nodes. 
 
 1. Identify a pool of IP addresses for the services running in K3s.
+
+   **Note**: The identification of a pool of IP addresses for the services running in K3s should be made at system installation time in order to avoid the possibility of IP collisions.
```

Comment:
We need this advice somewhere in the UAN install procedures.

Reply:
Actually, it needs to be in the install of the system docs, likely CSM, so the subnets are defined before CSM is installed.  If an admin/installer waits until after CSM is started, there is a risk of the upper half of the [CAN|CHN] Dynamic MetalLB subnet being used by CSM services which would end up with IP collisions for UAN K3s services.

Reply:
I added the info to the uan installation prereqs doc.